Core/HLE/sceNet.cpp

```diff
@@ -548,6 +573,11 @@ const HLEFunction sceNetUpnp[] = {
 	{0XE24220B5, &WrapI_II<sceNetUpnpInit>,          "sceNetUpnpInit",                  'i', "ii"   },
 };
 
+const HLEFunction sceNetIfhandle[] = {
+	{ 0xC80181A2, &WrapI_UU<sceNetGetDropRate>,     "sceNetGetDropRate",                 'i', "ii" },
```

Comment:
Should be: `'i', "pp" },` for `sceNetGetDropRate`.  This indicates that the parameters are pointers (which they are.)

`sceNetSetDropRate` is fine as is.

-[Unknown]